testdata/scripts/scopegen.txt

```diff
@@ -88,9 +88,9 @@ package test
 type AuthScope string
 
 const (
-	Scope_admin AuthScope = "Grants read and write access to administrative information",
-	Scope_read AuthScope = "Grants read access",
-	Scope_write AuthScope = "Grants write access",
+	Scope_admin AuthScope = "Grants read and write access to administrative information"
+	Scope_read AuthScope = "Grants read access"
+	Scope_write AuthScope = "Grants write access"
```

Comment:
could you add a step in this script to make sure the generated go file can compile? Otherwise, LGTM.

Reply:
@favadi i added an `exec go build all.scopes.go` cmd to check for the syntax. hope this solves it!